ios/NotificationService/NotificationService.swift

```diff
@@ -55,21 +55,22 @@ class NotificationService: UNNotificationServiceExtension {
     if #available(iOSApplicationExtension 15.0, *) {
       let overrideUsername = notification.userInfo["override_username"] as? String
       let senderId = notification.userInfo["sender_id"] as? String
-      let sender = overrideUsername ?? senderId
 
-      guard let serverUrl = notification.userInfo["server_url"] as? String,
-              let sender = sender
+      guard let serverUrl = notification.userInfo["server_url"] as? String
       else {
         os_log(OSLogType.default, "Mattermost Notifications: No intent created. will call contentHandler to present notification")
         self.contentHandler?(notification)
         return
```

Comment:
No completion before this return?

Reply:
contentHandler is a completionHandler but in this case for the entire service and not a specific function as the others 😃